src/main/java/org/mockito/testng/internal/MockitoBeforeTestNGMethod.java

```diff
@@ -50,9 +52,14 @@ private void initMocks(ITestResult testResult) {
         if (alreadyInitialized(testInstance)) {
             return;
         }
+
+        String strictnessValue = testResult.getTestContext().getSuite().getXmlSuite().getParameter(STRICTNESS_LEVEL);
```

Comment:
That may be wise to abstract away this `testResult.getTestContext().getSuite().getXmlSuite().getParameter(STRICTNESS_LEVEL)` in another testng _access_ class that returns an optional.

```java
Strictness strictness = TestNGAccess.getParameter(testResult, STRICTNESS_LEVEL).map(Strictness::valueOf).orElse(Strictness.STRICT_STUBS);
```

Reply:
Or maybe write `TestNGAccess.getStrictness(testResult).orElse(...)`, because this method can perform some validation on the value, and fail with a `MockitoException` if the value is not one of `Strictness` enum.

Reply:
Out of the options you suggested, I can proceed with detecting the invalid value before a test is executed and throwing a MockitoException.  That would be more user-friendly than proceeding with a default value and failing tests due to strict stubs.